packages/framework/assets/js/admin/components/FileItem.js

```diff
@@ -1,4 +1,4 @@
-import { forceValidateElement } from '../validation/customization/validation';
+import { forceValidateElement } from '../validation/customization/validationInit';
```

Comment:
Why is this changed? or why is not the `validation.js` removed at all. These files are the same except path in `import` part.

Reply:
374c67a886d912b7eaaaeec57089a93414dca4d8